esphome/components/sx1509/sx1509_keypad_sensor.cpp

```diff
@@ -0,0 +1,59 @@
+#include "sx1509_keypad_sensor.h"
+#include "esphome/core/helpers.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace sx1509 {
+
+static const char *TAG = "sx1509_keypad_sensor";
+
+void SX1509KeypadSensor::setup() {
+  ESP_LOGD(TAG, "setup   rows: %d , cols: %d", this->rows_, this->cols_);
```

Comment:
```suggestion
  ESP_LOGV(TAG, "setup   rows: %d , cols: %d", this->rows_, this->cols_);
```

Or make the message more user-readable.